### ISSUE TITLE
Edit Files: Disable save changes button the first time it is pushed to avoid multiple actions.

### ISSUE THREAD

**Issue by kcondon** [IQSS/dataverse#2545]
During testing and there has been evidence in production that leaving the save changes button enabled after it has been pressed can lead to inadvertently making multiple submits such as multiple draft versions of datasets, multiple file uploads of the same file, etc. 

Usually these issues happen when some error occurs on save: failed ingest, etc so fixing these secondary errors resolves the specific cause but it would better to resolve the root cause: disable the button on submit.

**Comment by landreev** [comment]
I'm not sure I can get this done today.

**Comment by landreev** [comment]
Rather than disabling the button on the page, I modified the backing bean so that only one save is performed. ONCE the version passes the validation that is - i.e., if it is an illegal character in the filename, or some other validation error, that the user can actually fix - they will be able too. 
But if the page gets stuck again in that mode where it's actually saving a new DRAFT version, but can't complete the remaining tasks - they will not be able to create multiple draft versions. 

I have to point out though, that the chances of this condition to reappearing are quite slim. Usually, if this page fails, it will fail actually trying to save the version... So, rather than being able to create multiple versions, they will not be able to create any. This behavior observed was a result of some exotic circumstances; it was able to save the version, but could not start the async. ingest jobs. And that really is the "remaining tasks" I mentioned above. I tried to make sure this was not going to happen again. And I don't think there's anything else there that can fail, after the version save. 

(for disabling the button - Gustavo has a nice solution, that greys out the page and shows a progress indicator... but it may not happen in 4.3 - he wanted to run it by Mike and Liz first).
